shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/model/schema/model/schema/SchemaMetaDataLoader.java

```diff
@@ -42,6 +42,8 @@
     
     private static final String TABLE_TYPE = "TABLE";
     
+    private static final String VIEW_TYPE = "VIEW";
```

Comment:
Why do we need `VIEW_TYPE`?

Reply:
> Why do we need `VIEW_TYPE`?

@tristaZero In order to support select view statement.